components/functions-library-mgt/org.wso2.carbon.identity.functions.library.mgt/src/main/java/org/wso2/carbon/identity/functions/library/mgt/FunctionLibraryManagementServiceImpl.java

```diff
@@ -168,10 +176,12 @@ private void evaluateScript(FunctionLibrary functionLibrary) throws FunctionLibr
             code = head + code;
             engine.eval(code);
         } catch (ScriptException e) {
-            log.error("Function library script of " + functionLibrary.getFunctionLibraryName() +
+            log.debug("Script library script of " + functionLibrary.getFunctionLibraryName() +
```

Comment:
check if debug is enabled

Reply:
My bad. It should be a log.error() not log.debug.

Reply:
[Fixed commit](https://github.com/wso2/carbon-identity-framework/pull/2679/commits/9a526929402c0b6fb5ef8eab5d87a89fbaeee7c8)